### ISSUE TITLE
logo

### ISSUE THREAD

**Issue by ahmeterbay** [lutris/lutris#872]
hi..
can i help you to design a new logo for your gaming platform?
it will be free!

**Comment by strycore** [comment]
Hi, what did you have in mind?

**Comment by ahmeterbay** [comment]
i have no idea for new one.. but we can change it if you wanna.. it can be better :)

**Comment by strycore** [comment]
I'm ok with the current one for the moment.

**Comment by strycore** [comment]
No wait, I've changed my mind. Sorry about closing the issue so promptly, I'm trying to keep the number of open issues as low as possible.

In addition to the logo we currently have it would be nice to have a logo we can have in a smaller icon, for example, a tray icon or the website's favicon. Our current logo is fine, but doesn't render very well under 24px since it's too complex.

A simpler logo would render much better at small sizes and would be very good to have! It doesn't necessarily have to be related to the current logo. Anyway, if you're still up for it, I'm open for suggestions!

![showmewhatyougot-1](https://user-images.githubusercontent.com/280336/41461363-60b3007a-7044-11e8-9556-8b71d9c1160a.jpg)

**Comment by AccountOneOff** [comment]
I like the otter. It's distinctive and it stays true to the name. What about these?

https://github.com/daniruiz/flat-remix/blob/master/Flat-Remix-Dark/apps/scalable/lutris.svg

https://github.com/PapirusDevelopmentTeam/papirus-icon-theme/blob/master/Papirus/64x64/apps/lutris.svg

Both are under the GPLv3 license.

**Comment by AccountOneOff** [comment]
Yeah that looks nice. What about requesting the Papirus team make a symbolic icon?

**Comment by basxto** [comment]
An additional possibility would be to request one at https://icons8.com/request-icon/

It’s a commercial project, but all their icons are licensed under cc-by-nd 3.0: https://icons8.com/license/
